samples/drivers/crypto/sample.yaml

```diff
@@ -27,3 +27,8 @@ tests:
         - ".*: CBC Mode"
         - ".*: CTR Mode"
         - ".*: CCM Mode"
+  sample.drivers.crypto.stm32:
+    tags: crypto
+    platform_whitelist: mikroe_mini_m4_for_stm32
```

Comment:
I would have preferred filter, so we can easily add more targets. 
Any reason why you preferred whitelist ?

Reply:
The filter used before didn't work (project was built on multiple targets and failed for all but the mikroe_mini_m4_for_stm32). Can you share the filter expression you have in mind?

Reply:
Actually, I would have expected the filter to work.
Let's stick with the white list for now.